substrate/frame/aura/src/lib.rs

```diff
@@ -363,7 +368,7 @@ impl<T: Config> FindAuthor<u32> for Pallet<T> {
 		for (id, mut data) in digests.into_iter() {
 			if id == AURA_ENGINE_ID {
 				let slot = Slot::decode(&mut data).ok()?;
-				let author_index = *slot % Self::authorities().len() as u64;
+				let author_index = *slot % Self::authorities_len() as u64;
```

Comment:
With the current changes (possibly even before this PR), this could be a division by 0 here. We could do a check and return no author if the authority count is 0.

Reply:
With `0` authorities you don't have anyone that can produce a block and thus, you would never end up here :P